### ISSUE TITLE
Impossible to pass generic Pandoc options in Rules

### ISSUE THREAD

**Issue by ddfreyne** [nanoc/nanoc#526]
It is impossible to pass these options to Pandoc in the Rules file:

```ruby
:s, {:f => :markdown, :to => :html}, 'no-wrap', :toc
```

**Comment by ghiknt** [comment]
After further experiment it is also impossible to pass options twice (e.g. --filter).  Since the last one is all that is kept in the hash.  Temporally worked around by adapting test case to filter wrapper:

```ruby
class PandocCommandLine < Nanoc::Filter
  identifier :pandoc_commandline
  type :text
  def run(content, params={})
   filter = ::Nanoc::Filters::Pandoc.new
   filter.setup_and_run(content, *params[:args])
  end
end
```

Which is called by Rules file.
```ruby
opts = ['table-of-contents', {:to => :html5, 'base-header-level' => 2}, {:filter =>'pandoc-zotxt'}, {:filter => 'pandoc-citeproc'}]
filter :pandoc_commandline, :args => opts 
```

* Note table-of-contents is not a good one to use for testing though since it only gets rendered in latest pandoc when --standalone is specified.

**Comment by ddfreyne** [comment]
Yeah, I feel that the only solution to this would be to introduce an arbitrary `:args` param, as you suggested. I think I’ll fix it like that; there’ll be a PR incoming soon.

**Comment by ddfreyne** [comment]
Potential fix in #535.

**Comment by ddfreyne** [comment]
Fixed by #535.
